students/dae-hwan/user/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from user.views import SignUpView
+
+urlpatterns = [
+    path('/sign_up', SignUpView.as_view())
```

Comment:
```suggestion
    path('/signup', SignUpView.as_view())
```
url에서는 '_'에 대신에 '-'으로 단어를 연결합니다. 그리고 signup 같은 경우 관용적으로 붙여서 사용합니다.